docs/applications/container-repositories/02-github.md

```diff
@@ -0,0 +1,133 @@
+          namespace: ${{ vars.NAMESPACE }}
+          image-name: ${{ vars.IMAGE_NAME }}
+          artifact: ${{ secrets.MASSDRIVER_ARTIFACT_ID }}
+          region: ${{ vars.REGION }}
+          image-tag: ${{ github.sha }}
+          build-context: ./
+      - name: Publish Bundle 
+        uses: massdriver-cloud/actions/bundle_publish@v3
+        with:
+          build-directory: ./
+      - name: Set Image Version 
+        uses: massdriver-cloud/actions/app_patch@v3
+        with:
+          project: <insert-project-name-here>
+          target: <insert-environment-name-here>
+          manifest: <insert-bundle/app-manifest-here>
+          set: |
+            <insert-image-tag-path> = "${{ github.sha }}"
+      - name: Deploy App
+        uses: massdriver-cloud/actions/app_deploy@v3
+        with:
+          project: <insert-project-name-here>
+          target: <insert-environment-name-here>
+          manifest: <insert-bundle/app-manifest-here>
+
+```
+
+You may need to modify the branch name in the `on: push:` section of the workflow to match the branch you're using for your application.
+
+When this GitHub Action runs, it will:
+* Build and push your image to your cloud container repository
+* Update the tag in your application package
+* Redeploy your application in Massdriver with the updated tag
+
+### Where can I find my secrets, project, target, and manifest names?
+
+* [Secrets & vars](#set-secrets-and-vars)
+* [Projects](/concepts/projects) are found on the Projects page of Massdriver
+* [Targets](/concepts/environments) are found listed inside your project
+* [Manifests](/concepts/manifests) are the abbreviated name of your bundle or application
+
+![Manifest Name](manifest.png)
+
+### Where can I find my image tag path?
+
+The image tag path is the path configured within your application's `massdriver.yaml` file. For example, if your `massdriver.yaml` file looks like this:
+
+```yaml title="massdriver.yaml"
+runtime:
+  type: object
+  title: Runtime Settings
+  required:
+    - image
+  properties:
+    image:
+      type: object
+      title: Container Image
+      required:
+        - tag
+      properties:
+        tag:
+          title: ECR image tag
+          description: Specific version of the container to pull on each deployment of the lambda function.
+          type: string
+          default: latest
+```
+Then your image tag path would be `.runtime.image.tag`. 
+
+### I have subdirectories in my repository. How do I set the build context?
+
+* If your `Dockerfile` is in a subdirectory, you can update the `build-context` to point to that directory. For example, if your `Dockerfile` is in the `./app` directory, you can set the `build-context` to `./app`.
+* If your `massdriver.yaml` file is in a subdirectory, you can update the `build-directory` to point to that directory. For example, if your `massdriver.yaml` file is in the `./app/massdriver` directory, you can set the `build-directory` to `./app/massdriver`.
```

Comment:
This is great!  Let's assume people might not read this, and update the example action to have <path to massdriver.yaml directory> or something. That increases the chance they'll read further to see "what do they mean by path to massdriver.yaml" and find this.